src/components/Company/Industry/IndustrySelect.test.tsx

```diff
@@ -12,9 +14,11 @@ describe('IndustrySelect', () => {
     vi.mocked(loadAll).mockResolvedValue([{ code: 'abcd', title: 'Do Things' }])
 
     render(
-      <ThemeProvider>
-        <IndustrySelect />
-      </ThemeProvider>,
+      <ComponentsProvider value={defaultComponents}>
```

Comment:
Nit: should this just be updated to use the gusto test provider instead? With that update this would've gotten the adapter changes/context automatically